### PR TITLE
Meta tag Description,Keywords

### PR DESCRIPTION
**Pull request by Saichethan**
### Description
Added Meta tags "Description & Keywords" to all pages except old.index.html

Fixes # [ISSUE]
#95 

- Code
- Quality Assurance
- User Interface

  

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -134,7 +134,7 @@ <h3>Google Summer of Code</h3>
 
     <h3>Outreachy (winter rounds only)</h3>
         <p class="years">
-            <a href="/outreachy/2017.html">2017</a>
+            <a href="/Outreachy/2017.html">2017</a>
```
Comment (shreyateeza) [inline index.html]: @Saichethan Please create a separate issue and PR requiring this change. Actually there are two different kind of changes in the same PR. Hope you understand!

File: newcomers.html [hunk 2]
```diff
@@ -4,6 +4,9 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <title>Systers Open Source Projects by Systers</title>
+    
+    <meta name="Description" content="Very new to the community. Every Beginner needs to update their profile within 30 days of joining Systers Open Source Slack or they will be removed.">
+    <meta name="Keywords" content="New Comers,Systers,new comers, getting started with systers,AnitaB.org">
```
Comment (shreyateeza) [inline newcomers.html]: @Saichethan Please write 'Newcomers' instead of 'New comers' and so on.

**Comment by Saichethan** [comment]
@shreyateeza I think I am done! Correct me if I am wrong

**Comment by Saichethan** [comment]
I previously implemented meta tags & schema for student club website if my college & it worked fine so, I took it as granted let me know if I am wrong

**Comment by sara-02** [comment]
@Saichethan As we have been pointing out, content is good, but you need to perform git squash operation, and update the PR description with appropriate content.

**Comment by sara-02** [comment]
@Saichethan https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by sara-02** [comment]
@Saichethan After performing git squash successfully from this branch, you will need to do `git push -f origin <branch_name>`. I hope this clears the doubt.

**Comment by AnkitKhatri55555** [comment]
@Saichethan , you need to fill the empty sections you left in the PR template.As far as your content change is concerned , its fine.And perform git squash to squash unnecessary commits and only pick those pertaining to the issue

**Comment by AnkitKhatri55555** [comment]
You should make new branch for each issue, your master branch is 5 commits ahead, so you can fix your master branch first  (https://stackoverflow.com/questions/9646167/clean-up-a-fork-and-restart-it-from-the-upstream)  and then update PR's

**Comment by MeepyMay** [comment]
Was not assigned issue. Nor was the issue approved. Closing PR.

**Pull request closed without merging**